ucsmsdk/ucscoreutils.py

```diff
@@ -57,11 +57,7 @@ def get_ucs_obj(class_id, elem, mo_obj=None):
         mo_class_params = inspect.getargspec(mo_class.__init__)[0][2:]
         mo_class_param_dict = {}
         for param in mo_class_params:
-            mo_param = mo_class.prop_meta[param].xml_attribute
-            if mo_param not in elem.attrib:
-                mo_class_param_dict[param] = ""
-                continue
-            mo_class_param_dict[param] = elem.attrib[mo_param]
+            mo_class_param_dict[param] = None
```

Comment:
@ragupta-git where do we set the mo params? If I am reading the code right - then the `**mo_class_params_dict`` which is passed as kwargs to `mo_class` will ave all property values as `None`. and the other changes take care of setting `rn` and `dn`. What about other properties?